src/Core/ServiceWrapper/Program.cs

```diff
@@ -658,6 +659,39 @@ private static string ReadPassword()
             }
         }
 
+        private static IWinSWConfiguration GetConfigType()
+        {
+            var executablePath = new DefaultWinSWSettings().ExecutablePath;
+            var baseName = Path.GetFileNameWithoutExtension(executablePath);
+
+            if (baseName.EndsWith(".vshost"))
```

Comment:
You could also remove this block by the way. This should not be a part of the main logic.

Reply:
Sorry, I don't get it? Don't we need to remove .vshost extension before start searching the file?

Reply:
No.

Reply:
It makes sense to keep it for XML though

Reply:
This was only for testing. The test host today is not *.vshost.exe*.